source/compatibility.go

```diff
@@ -123,6 +125,11 @@ func legacyEndpointsFromDNSControllerService(svc *v1.Service, sc *serviceSource)
 // It will use node role label to check if the node has the "node" role. This means control plane nodes and other
 // roles will not be used as targets.
 func legacyEndpointsFromDNSControllerNodePortService(svc *v1.Service, sc *serviceSource) ([]*endpoint.Endpoint, error) {
+	if sc.nodeInformer == nil {
+		log.Warnf("Unable to extract nodePort targets from service %s/%s as nodePort support is disabled", svc.Namespace, svc.Name)
```

Comment:
Since the user explicitly disabled nodePort support then Info seems more appropriate than Warn.
```suggestion
		log.Infof("Unable to extract nodePort targets from service %s/%s as nodePort support is disabled", svc.Namespace, svc.Name)
```
I've suggested Info level here (and elsewhere Warn is used) but I see other places in the PR where Debug is used.

Reply:
I do not thinkg it's an ifo level, but not a warning as well. What this should warn us about?

**log.Warn**:
    - Purpose: Indicates a potential issue or a situation that might lead to problems in the future.
    When to Use:
        - Unexpected Behavior: When something unexpected happens but the application can still function.
        - Resource Limitations: When resources are running low (e.g., disk space, memory).
        - Configuration Issues: When encountering unexpected or invalid configuration values.

**log.Debug**:

    - Purpose: Used for detailed information that is primarily helpful for debugging.
    When to Use:
        - Development and Testing: Useful for tracking the flow of execution, inspecting variable values, and understanding the internal workings of your application.
        - Troubleshooting: Helps pinpoint the root cause of issues during development and debugging.

Do you think there is a value in having this log as a warning instead of a debug?

Reply:
The message contains `as nodePort support is disabled`

Is it for cases when `nodePort` explicitly disabled or accidently and RBAC needs modification?